user.js

```diff
@@ -0,0 +1,8 @@
+
+export const defaultUser = {
+    username : 'jackrrr123',
+    name : 'Jack Rowsell',
+    avatar_url: 'https://i.pinimg.com/736x/fa/39/eb/fa39eb88592bf16aebab746f88068ac7.jpg'
+}
```

Comment:
Watch out using this as the default user - it doesn't actually exist in the db so wont work for posting comments :)